cmd/lotus-seal-worker/main.go

```diff
@@ -0,0 +1,91 @@
+package main
+
+import (
+	"github.com/mitchellh/go-homedir"
+	"os"
+
+	logging "github.com/ipfs/go-log"
+	"golang.org/x/xerrors"
+	"gopkg.in/urfave/cli.v2"
+
+	"github.com/filecoin-project/lotus/api"
+	"github.com/filecoin-project/lotus/build"
+	lcli "github.com/filecoin-project/lotus/cli"
+)
+
+var log = logging.Logger("main")
+
+func main() {
+	logging.SetLogLevel("*", "INFO")
+
+	log.Info("Starting lotus worker")
+
+	local := []*cli.Command{
+		runCmd,
+	}
+
+	app := &cli.App{
+		Name:    "lotus-seal-worker",
+		Usage:   "Remote storage miner worker",
+		Version: build.Version,
+		Flags: []cli.Flag{
+			&cli.StringFlag{
+				Name:    "repo",
+				EnvVars: []string{"WORKER_PATH"},
+				Value:   "~/.lotusworker", // TODO: Consider XDG_DATA_HOME
+			},
+			&cli.StringFlag{
+				Name:    "storagerepo",
+				EnvVars: []string{"LOTUS_STORAGE_PATH"},
+				Value:   "~/.lotusstorage", // TODO: Consider XDG_DATA_HOME
+			},
+		},
+
+		Commands: local,
+	}
+
+	if err := app.Run(os.Args); err != nil {
+		log.Warnf("%+v", err)
+		return
+	}
+}
+
+var runCmd = &cli.Command{
+	Name:  "run",
+	Usage: "Start lotus worker",
+	Action: func(cctx *cli.Context) error {
+		nodeApi, closer, err := lcli.GetStorageMinerAPI(cctx)
```

Comment:
Does this mean it can't easily be run on a separate computer?

Reply:
You would probably need a fake path with just the `api` file.

Reply:
^ yes, can make this nicer later